modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc

```diff
@@ -33,218 +46,424 @@ After {cgu-operator} completes a cluster update, the cluster does not update aga
 * When the cluster changes to non-compliant with the `inform` policy after being updated
 ====
 
-[id="upgrade_not_started"]
-== The UpgradeNotStarted state
+[id="selecting_clusters"]
```

Comment:
[Every ID in a module](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files) needs the `_{context}` variable added, e.g.:

```suggestion
[id="selecting_clusters_{context}"]
```

Reply:
thanks. added `_{context}` to each ID